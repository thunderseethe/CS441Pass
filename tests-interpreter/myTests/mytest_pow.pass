//Test for pow operator (^)

begin
	print (2 ^ 3);
	int x = 4;
	float y = 0.5;
	print (x ^ y);
	print (2.5 ^ 2);
end