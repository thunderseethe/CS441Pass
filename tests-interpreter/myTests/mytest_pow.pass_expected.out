8
2.000000
6.250000
